linux-bore/PKGBUILD

```diff
@@ -248,39 +248,22 @@ if [ -n "$_latency_nice" ]; then
          source+=("${_patchsource}/misc/0001-Add-latency-priority-for-CFS-class.patch")
     fi
 fi
-## BMQ Scheduler
-if [ "$_cpusched" = "bmq" ]; then
-    source+=("${_patchsource}/sched/0001-prjc-cachy.patch")
-fi
-## PDS Scheduler
-if [ "$_cpusched" = "pds" ]; then
-    source+=("${_patchsource}/sched/0001-prjc-cachy.patch")
-fi
-## BORE Scheduler with latency_nice
-if [ "$_cpusched" = "bore" ]; then
-    source+=("${_patchsource}/sched/0001-bore-cachy.patch")
-## BORE SYSCTL TUNING
-   if [ -n "$_tune_bore" ]; then
-		source+=("${_patchsource}/misc/0001-bore-tuning-sysctl.patch")
-   fi
-fi
-## CacULE Scheduler
-if [ "$_cpusched" = "cacule" ]; then
-    source+=("${_patchsource}/sched/0001-cacULE-cachy.patch")
-fi
-## CacULE-RDB Scheduler
-if [ "$_cpusched" = "cacule-rdb" ]; then
-    source+=("${_patchsource}/sched/0001-cacULE-cachy.patch")
-fi
-## TT Scheduler
-if [ "$_cpusched" = "tt" ]; then
-    source+=("${_patchsource}/sched/0001-tt-cachy.patch")
-fi
-## Hardened Patches with BORE Scheduler
-if [ "$_cpusched" = "hardened" ]; then
-    source+=("${_patchsource}/sched/0001-bore-cachy.patch"
-         "${_patchsource}/misc/0001-hardened.patch")
-fi
+
+case "$_cpusched" in
+    pds|bmq) # BMQ/PDS scheduler
+        source+=("${_patchsource}/sched/0001-prjc-cachy.patch");;
+    tt) ## TT Scheduler
+        source+=("${_patchsource}/sched/0001-tt-cachy.patch");;
+    cacule|cacule-rdb) ## CacULE(-RDB) Scheduler
+        source+=("${_patchsource}/sched/0001-cacULE-cachy.patch");;
+    bore) ## BORE Scheduler with latency_nice
+        [ -n "$_tune_bore" ] && source+=("${_patchsource}/misc/0001-bore-tuning-sysctl.patch")
+        source+=("${_patchsource}/sched/0001-bore-cachy.patch");;
+    bore-hardened) ## Hardened Patches with BORE Scheduler
```

Comment:
In original condition, the bore-tuning-sysctl patch was also not applying for hardened. Is that normal?

Reply:
I need to check if it is applying well, otherwise we can add it

Reply:
OK.